lib/gtfs/hastus/schedule.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Gtfs.Hastus.Schedule do
+  @moduledoc """
+  A HASTUS schedule id refers to the same idea as a GTFS service id:
+  a particular version of service that determines what trips run on a day:
+  But the ids are incompatible between HASTUS and GTFS,
+  so they get different types so it's clear they aren't interchangable.
+  """
+  @type id :: String.t()
```

Comment:
How are we using Schedule IDs?

Reply:
They're part of the unique key (`{schedule_id, run_id}` or `{schedule_id, block_id}`) that lets us look up data for the minischedules.

Reply:
Is that problematic if they are incompatible between HASTUS and GTFS? Are we able to export the GTFS version from HASTUS?

Reply:
The GTFS version is made by some complex logic in `gtfs_creator`, and doesn't exist in HASTUS.

We'll be able to look the schedule_id up by the trip_id, in the same way as we look up the service_id by the trip_id so we can fetch blocks in vehicle.ex.